packages/web-client/app/helpers/config.ts

```diff
@@ -0,0 +1,24 @@
+// Adapted from https://github.com/nvdk/ember-config-helper/blob/6a9891afce1ef68c1a42fb5e5fc54ce02f5ae811/addon/helpers/config.js
+import { getOwner } from '@ember/application';
+import Helper from '@ember/component/helper';
+import { get } from '@ember/object';
+
+export default class extends Helper {
+  config: any | undefined;
+
+  constructor() {
+    super(...arguments);
+
+    this.config = getOwner(this)
+      .resolveRegistration('config:environment')
+      .clone();
+
+    this.config.unknownProperty = (path: string) => {
+      throw new Error(`Unknown config property: ${path}`);
+    };
```

Comment:
I added the `clone` above to avoid attaching `unknownProperty` to the global-esque `config:environment`. Is it even worth it? It just seemed dirty to me to mutate that upon instantiation of this helper.

Reply:
lol `clone` doesn’t exist… now I have removed that, but if mutation-avoidance seems important, `JSON.parse(JSON.stringify` is one option, sadly hideous though.

Reply:
`unknownProperty` doesn't seem to be called with missing, nested properties. Maybe the nested object that is missing the property needs to also implement `unknownProperty`? 

I think it's probs better to throw in the compute function if it's unknown, given this limitation.

Reply:
wow, huge oversight, thanks for catching that. Do you mean that it should throw if the `get` returns `null`?

Reply:
> Now I’m wondering if linting might work here, to check against properties known to exist in config 🤔

Oh that sounds even better for non-dynamic (I hope it's always static, dynamic strings for this sounds messy) cases!